encoding/wkb/wkb.go

```diff
@@ -245,84 +237,77 @@ func (d *Decoder) Decode() (orb.Geometry, error) {
 	return nil, ErrUnsupportedGeometry
 }
 
-func readByteOrderType(r io.Reader, buf []byte) (byteOrder, uint32, error) {
+func readByteOrderType(r io.Reader, buf []byte) (binary.ByteOrder, uint32, error) {
 	// the byte order is the first byte
 	if _, err := r.Read(buf[:1]); err != nil {
-		return 0, 0, err
+		return DefaultByteOrder, 0, err
 	}
 
-	var order byteOrder
+	var order binary.ByteOrder
 	if buf[0] == 0 {
-		order = bigEndian
+		order = binary.BigEndian
 	} else if buf[0] == 1 {
-		order = littleEndian
+		order = binary.LittleEndian
 	} else {
-		return 0, 0, ErrNotWKB
+		return DefaultByteOrder, 0, ErrNotWKB
 	}
 
 	// the type which is 4 bytes
 	typ, err := readUint32(r, order, buf[:4])
 	if err != nil {
-		return 0, 0, err
+		return DefaultByteOrder, 0, err
```

Comment:
I believe this is actually a change, previously it would return `0` (bigEndian) and now it returns `DefaultByteOrder` (littleEndian).

Probably not a big deal since its an error case where presumably the value is ignored, just wanted to mention it.

Reply:
note: theres a few cases of this